src/modules/webhooks/transaction-initialize-session.test.ts

```diff
@@ -24,10 +24,10 @@ describe(`TransactionInitializeSessionWebhookHandler`, () => {
           "x-stripe-client-user-agent",
         ],
       },
-      method: false,
-      body: false,
-      order: false,
-      url: false,
+      method: true,
+      body: true,
+      order: true,
```

Comment:
Does the order really matter in recordings? 🤔

Reply:
You're right, it shouldn't